LSN_10/home-work/main.js

```diff
@@ -0,0 +1,162 @@
+const header = document.getElementById('header')
+
+const container = document.getElementById('cont')
+header.appendChild(container)
+
+console.log(container);
+
+const profile = document.getElementById('profile')
+container.appendChild(profile)
```

Comment:
everything up to this line is redundant. You already have these containers that are nested inside each other